src/components/StylesFromLiteralsService.js

```diff
@@ -104,16 +104,40 @@ goog.provide('ga_stylesfromliterals_service');
           var values = properties.values;
           for (var i = 0; i < values.length; i++) {
             var value = values[i];
-            olStyle = getOlStyleFromLiterals(value);
-            this.styles[value.geomType][value.value] = olStyle;
+            var minResolution = value.minResolution !== undefined ?
+                value.minResolution : 0;
+            var maxResolution = value.maxResolution !== undefined ?
+                value.maxResolution : Infinity;
+            olStyle = {
+              olStyle: getOlStyleFromLiterals(value),
+              minResolution: minResolution,
+              maxResolution: maxResolution
+            };
+            if (!this.styles[value.geomType][value.value]) {
+              this.styles[value.geomType][value.value] = [olStyle];
+            } else {
+              this.styles[value.geomType][value.value].push(olStyle);
+            }
           }
         } else if (this.type === 'range') {
```

Comment:
The content of the 2 `else if` seems very close. any chance to avoid this duplication of code ?